modules/tinymce/src/plugins/table/main/ts/api/Api.ts

```diff
@@ -13,21 +13,18 @@ import { HTMLElement } from '@ephox/dom-globals';
 import { ResizeHandler } from '../actions/ResizeHandler';
 import { SelectionTargets } from '../selection/SelectionTargets';
 
-const getClipboardRows = (clipboardRows): HTMLElement[] => clipboardRows.get().fold(function () {
-  return;
-}, function (rows) {
-  return Arr.map(rows, function (row) {
-    return row.dom();
-  });
-});
+const getClipboardRows = (clipboardRows): HTMLElement[] => clipboardRows.get().fold(
+  () => {},
+  (rows) => Arr.map(rows, (row) => row.dom())
+);
 
 const setClipboardRows = (rows: HTMLElement[], clipboardRows) => {
   const sugarRows = Arr.map(rows, Element.fromDom);
   clipboardRows.set(Option.from(sugarRows));
 };
 
 const getApi = (editor: Editor, clipboardRows: Cell<Option<any>>, resizeHandler: ResizeHandler, selectionTargets: SelectionTargets) => ({
-  insertTable: (columns: number, rows: number) => InsertTable.insert(editor, columns, rows),
+  insertTable: (columns: number, rows: number) => InsertTable.insert(editor, columns, rows, 0, 0),
```

Comment:
Actual feature stuff - this is meant to open the dialog, hence default header args to 0. Maybe should be different?

Reply:
Possibly. I couldn't decide if we wanted to change this API or not. Consistency is probably important here though, yeah

Reply:
- [ ] Allow args to be passed through

Reply:
just realised - should this use the same kind of options object as the command?

Reply:
Good point... I guess so, as that'd keep it consistent :shrug: